package.json

```diff
@@ -43,16 +43,17 @@
     "pretty-quick": "3.1.3",
     "style-loader": "3.3.3",
     "svgo-loader": "4.0.0",
+    "ts-jest": "29.1.1",
```

Comment:
can we re-order the dependencies so that only the new one `buffer` shows up in the diff?